### PR TITLE
registry package pages

### PR DESCRIPTION
**Pull request by sean1588**
theme: https://github.com/pulumi/theme/pull/397

issues:
https://github.com/pulumi/registry/issues/1531 - create package card
https://github.com/pulumi/registry/issues/1705 - compact card alerts
https://github.com/pulumi/registry/issues/1702 - H1s for API Docs
https://github.com/pulumi/registry/issues/1703 - H1s match selected tabs

figma designs: 
https://www.figma.com/file/bVEoSwAvU7KDpuPNri332F/Compact-Registry?node-id=117%3A3379&t=aHUxQCTiyRaFs11P-0
https://www.figma.com/file/bVEoSwAvU7KDpuPNri332F/Compact-Registry?node-id=227%3A5397&t=3sCE2WZGUSX9Dm5q-0

### CODE REVIEW

**Comment by anita-trimbur** [comment]
aahhh so exciting!! so glad to see this come together

a few minor style nits for the card itself: 
* the font and size of every element on the open card should be Inter 12px
* the only font that's different is the "last published" on the collapsed card which is Inter 10px gray600`#5A5A5A`
* (this is my bad, it's wrong in figma) we should use blue600 `#4D5BD9` for that info icon to match the note containers
* the info icon and the github icon should be the same size -- i think without the padding it'll be about 15x15
* i'm pretty sure the border color is a little light on the open card, should be gray300 `#DCDCDC`. on the closed card it's gray400 `#B0B0B0` along with the down/up chevron

**Comment by sean1588** [comment]
awesome. thanks @anita-trimbur. I'll get those changes made!

**Comment by anita-trimbur** [comment]
wow you got those changes in fast! 

last nit: can we shrink that external link icon by the source code repo? maybe down to like 10x10. i think that link text can also be 12px.

**Comment by anita-trimbur** [comment]
following up on our team discussion regarding grays. here's the breakdown

grays

tailwind is using the blue-grays that we were trying to get rid of. the _colors.scss file nearly matches the service grays in figma with the exception of gray 200, so it shouldn't be problematic to align gray200 as `#EFEFEF`. i think the difference in border color on the collapsed card was a color sampling error i made in figma -- sean you're right that it should match gray300 as it is defined in the service/.scss file.

**Review by anita-trimbur — APPROVED** [review]
🚀🚀🚀

File: themes/default/layouts/partials/registry/package/package-alert.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{{ $package := .Scratch.Get "package" }}
```
Comment (kimberleyamackenzie) [inline themes/default/layouts/partials/registry/package/package-alert.html:1]: TIL about scratch!

**Review by kimberleyamackenzie — APPROVED** [review]
Amazing!! 🎉 🚢 :shipit:

**Pull request merged**